packages/blueprints-integration/src/previews.ts

```diff
@@ -19,6 +23,55 @@ export enum PreviewType {
 	BlueprintImage = 'blueprintImage',
 }
 
+// The PreviewContent types are a partly replica of the types in PreviewPopUpContext.tsx
+export type PreviewContent =
+	| {
+			type: 'iframe'
+			href: string
+			postMessage?: any
+			dimensions?: { width: number; height: number }
+	  }
+	| {
+			type: 'image'
+			src: string
+	  }
+	| {
+			type: 'video'
+			src: string
+	  }
+	| {
+			type: 'script'
+			script?: string
+			firstWords?: string
+			lastWords?: string
+			comment?: string
+			lastModified?: number
+	  }
+	| {
+			type: 'title'
+			content: string
+	  }
+	| {
+			type: 'inOutWords'
+			in?: string
+			out: string
+	  }
+	| {
+			type: 'layerInfo'
+			layerType: SourceLayerType
+			text: Array<string>
+			inTime?: number | string
+			outTime?: number | string
+			duration?: number | string
```

Comment:
What do these properties mean? What's the difference between `outTime` vs `inTime + duration`? I'm also somewhat unsure about making this interface this loose & bound to a particular implementation of the hoverscrub preview component/any other component using this infromation - say Presenters' Screen. any change there would effectively require a change to blueprints integration, which we expect to be at least reasonably stable.

I'm also not a big fan of introducing terms `inTime` and `duration` - I think they somewhat overlap with `expectedStart` and `expectedDuration`? Are they the same? If that's the case, I think it would make sense to use the same names.

Reply:
Comments for it's use has been added to code.